Automattic/jetpack
# Linting: Check that project names are not duplicated

#### Changes proposed in this Pull Request:

For example, if projects/plugins/backup/composer.json and
projects/packages/backup/composer.json were to both be named
"automattic/jetpack-backup", that would confuse things.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Is CI happy?
* Edit `projects/plugins/backup/composer.json` to rename it to "automattic/jetpack-backup", then run the script. An error should be reported.
